src/main/webapp/resources/js/components/project/RemoteProjectStatus.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
```

Comment:
File comments

Reply:
Updated in 63bdc79ce23362f910cfd9f4b7843d4c10ff0743